src/TodoListItem.module.css

```
@@ -0,0 +1,96 @@
+:root {
+  --background-color: #cde9ff;
+  --secondary-color: #e4f3ff;
+  --main-font: "Courier New", Courier, monospace;
+  --main-font-color: #00335ad3;
+  --transition: ease-in-out 125ms;
+  --border-radius: 4px;
+  --box-shadow: 0px 1px 1px;
+}
+
+.Container {
+  padding: 20px;
+  text-align: center;
+  max-width: 900px;
+  margin: 20px auto;
+}
+.MainHeader {
+  color: var(--main-font-color);
+  font-size: 44px;
+}
+
+.Loading {
+  color: var(--main-font-color);
+  font-family: var(--main-font);
+}
+
+.FormInputField {
+  padding: 12px;
+  margin: 8px 8px 18px;
+  font-size: large;
+  background: var(--secondary-color);
+  border-radius: var(--border-radius);
+  font-family: var(--main-font);
+  border: 1px solid var(--background-color);
+  color: var(--main-font-color);
+  box-shadow: var(--box-shadow);
+}
+
+button {
+  background-color: var(--secondary-color);
+  font-family: var(--main-font);
+  border: 1px solid black;
+  border-color: var(--background-color);
+  border-radius: var(--border-radius);
+  color: var(--main-font-color);
+  box-shadow: var(--box-shadow);
+  padding: 15px;
+}
+
+button:hover {
+  cursor: pointer;
+  background-color: grey;
+  color: var(--background-color);
+  transition: var(--transition);
+}
+
+.ListItem {
+  padding: 10px;
```

try making each item a flexbox element too, then you'll have access to a couple of cool new commands that can do the spacing and the alignment like you were doing with `text-align` and the empty string `" "` before:
```suggestion
  padding: 10px;
  display: flex;
  gap: 10px;
```
flexbox has a left alignment by default (you could also try some settings like `justify-content: space-between` to see some other ways it can work.

then `gap` will act like a mergin between every one of your items inside of the flexbox element! super handy when you want to maintain the same spacing between several things and not add a bunch of extra classes to the html

---

sorry the suggestion is telling you to add a remove the same line, for the padding. I tried to leave that the same but I must've done something wrong in the formatting and I can't figure out what XD